### ISSUE TITLE
helm v2.15 is busted

### ISSUE THREAD

**Issue by sdake** [istio/istio.io#5221]
The docs should state to skip Helm v2.15.  See: https://github.com/istio/istio/issues/18095

**Comment by KnotGud** [comment]
I've been following this issue with a Kubernetes v1.16.2 install and noticed I could not install istio with either version (v1.15.0 and v1.14.3). Doing some reading, [Kubernetes removed API routes](https://kubernetes.io/docs/setup/release/notes/#deprecations-and-removals) `extensions/v1beta1` and moved them into `apps/v1` starting v1.16.0.

```bash
#helm v2.14.3
helm init --dry-run --debug > helm.yaml
```
Looking at the output of the command above, helm v2.14.3 seems to use `Deployment` in `extensions/v1beta1`. Helm also says in their [release notes](https://github.com/helm/helm/releases/tag/v2.15.0) for helm v2.15.0 they added support for future versions of Kubernetes i.e. changing to `apps/v1`.

tldr: Helm v2.14.3 can't be used for Kubernetes clusters v1.16

**Comment by howardjohn** [comment]
https://github.com/helm/helm/releases/tag/v2.15.1
